index.html

```diff
@@ -54,69 +61,103 @@ <h6>LET'S CONNECT</h6>
                     <div class="year">2015</div>
                 </div>
-                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <div class="details1">
+                    <div class="name">FACEBOOK</div>
+                    <div class="type">Full Stack Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description2">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <P class="description-2">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</P>
                 <ul class="tools">
                     <li class="html">html</li>
+                    <li class="ruby">Ruby on rails</li>
                     <li class="css">css</li>
                     <li class="javascript">javascript</li>
                 </ul>
-                <button class="see-project" type="button">See Project</button>
+                <button class="see-project2" type="button">See Project</button>
             </div>
-            <div class="sample-3">
-                <img src="./assets/images/Snapshoot-Portfolio-2.png" alt="image">
+            <div class="work-sample-3">
+                <img src="./assets/images/Snapshoot-Portfolio-2.png" alt="image" class="mobile-3">
+                <img src="./assets/images/SnapshootPortfolio2.png" alt="image" class="desktop-3">
                 <h3 class="title">Tonic</h3>
+                <h3 class="title1">Facebook 360</h3>
                 <div class="details">
                     <div class="name">CANOPY</div>
                     <div class="type">Back End Dev</div>
                     <div class="year">2015</div>
                 </div>
-                <P class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <div class="details1">
+                    <div class="name">FACEBOOK</div>
+                    <div class="type">Full Stack Dev</div>
+                    <div class="year">2015</div>
+                </div>
+                <P class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <P class="description-3">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</P>
                 <ul class="tools">
                     <li class="html">html</li>
+                    <li class="ruby">Ruby on rails</li>
                     <li class="css">css</li>
                     <li class="javascript">javascript</li>
                 </ul>
-                <button class="see-project" type="button">See Project</button>
+                <button class="see-project3" type="button">See Project</button>
             </div>
```

Comment:
- [ ] kindly, consider making a few alignments to your arrow icon, the 3 must be on the same line
<img width="622" alt="arr" src="https://user-images.githubusercontent.com/79058364/155741352-0d1fecef-d060-42ce-93be-e060747ec040.png">

Reply:
resolved 👍